docs/FAQ.md

```diff
@@ -0,0 +1,12 @@
+## FAQ
+
+Q: How to disable some plugins?
+A: You can disable one or more plugins by following methods.
+
+```python
+from skywalking import config
+
+config.disable_plugins = ['sw_http', 'sw_request']  # can be also CSV format, i.e. 'sw_http,sw_request'
```

Comment:
Where should user find these names？

Reply:
> Where should user find these names？

Added a section to list all the plugin names